helevents/templates/rest_framework/event_list.html

```diff
@@ -354,6 +354,39 @@ <h3 id="enrolment">Open enrolment</h3>
         <pre><code>event/?enrolment_open_waitilist=true</code></pre>
         <p><a href="?enrolment_open_waitilist=true" title="json">See the result</a></p>
```

Comment:
I know that this isn't part of the PR's scope, but mind fixing this while you're at it?
```suggestion
        <pre><code>event/?enrolment_open_waitlist=true</code></pre>
        <p><a href="?enrolment_open_waitlist=true" title="json">See the result</a></p>
```